futurex_openedx_extensions/dashboard/views.py

```diff
@@ -445,6 +445,14 @@ def get(self, request: Any, *args: Any, **kwargs: Any) -> JsonResponse:  # pylin
         return JsonResponse(get_tenants_info(tenant_ids))
 
 
+class AccessibleTenantsInfoViewV2(FXViewRoleInfoMixin, AccessibleTenantsInfoView):
```

Comment:
Please copy the docs from the older API and fix the description (anonymous access is not allowed for the new API). Also add a deprecation note to the older API, to be removed in version `v0.11.0.1`
```suggestion
@docs('AccessibleTenantsInfoViewV2.get')
class AccessibleTenantsInfoViewV2(FXViewRoleInfoMixin, AccessibleTenantsInfoView):
```